packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -901,3 +942,46 @@ class _FocusMarker extends InheritedNotifier<FocusNode> {
         assert(child != null),
         super(key: key, notifier: node, child: child);
 }
+
+/// A widget that controls whether or not the descendants of this widget are
+/// focusable.
+///
+/// Does not affect the value of [canRequestFocus] on the descendants.
+///
+/// See also:
+///  - [Focus], a widget for adding and managing a [FocusNode] in the widget tree.
+///  - [FocusTraversalGroup], a widget that groups widgets for focus traversal,
+///    and can also be used in the same way as this widget by setting its
+///    `descendantsAreFocusable` attribute.
+class ExcludeFocus extends StatelessWidget {
+  /// Const constructor for [ExcludeFocus] widget.
+  ///
+  /// The [descendantsAreFocusable] argument must not be null.
+  ///
+  /// The [child] argument is required, and must not be null.
+  const ExcludeFocus({
+    Key key,
+    this.descendantsAreFocusable = false,
```

Comment:
For consistency with ExcludeSemantics, IgnorePointer, etc. should this argument be called excluding?

Reply:
Sure, I went back and forth on that too.

I ended up here because I thought it more descriptive of what happened, because I could use the doc template, and because it has the opposite sense from `descendantsAreFocusable`, but I'm fine with being consistent. Renamed, swapped the default to "excluding = true", and expanded out the docs, correcting the sense of the flag in the docs.